builders.md

```diff
@@ -14,9 +14,9 @@ NOTE: This information can also be found in [network_information.md](network_inf
 | System | HPE DL325 |
 | lom0 Network | 10.12.2.252 & 2606:4100:3880:3102::252 |
 | IPMI Network | builder-00.ipmi.ofa.iol.unh.edu & 10.12.1.252 & 2606:4100:3880:3101::252 |
-| ROCE Fabric | Mellanox MCX653106A-HDAT Port 1 |
-| iWARP Fabric | Mellanox MCX653106A-HDAT Port 1 |
-| IB Fabric | Mellanox MCX653106A-HDAT Port 2 |
+| ROCE Fabric | Mellanox MCX653106A-HDAT Port 1 *(Connected to Mellanox QM8700 IB switch port 22)* |
```

Comment:
This doesn't make any sens, why is the ROCE / iWARP port connected to the infiniband switch?

Reply:
This was an error; the ROCE/iWARP ports are connected to **Edgecore Wedge 100BF-32x Ethernet switch port 22**.
This has been corrected [here].(https://github.com/OpenFabrics/fsdp_docs/pull/53/commits/5e2c2203ed485a421458c0715caa654b590bb8c3)